test/api/parameterizer.spec.js

```diff
@@ -0,0 +1,99 @@
+var assert = require('assert'),
+    sinon = require('sinon');
+
+var parameterizer = require('../../lib/dictionaries/api/parameterizer');
+
+
+
+describe('Parameterizer', function() {
+
+    var createMockResponse = function() {
+        return {
+            send: function() {},
+            status: function() {},
+            sendStatus: function() {}
+        }
+    };
+
+    it('should be a function', function() {
+        var p = parameterizer().default;
+        assert.equal('function', typeof p);
+    });
+
+    it('should invoke next when a users scope is passed', function() {
+
+        var req = {
+            params: {
+                scope: 'users'
+            }
+        };
+        var statusSet;
+        var res = {
+            sendStatus: function (status) {
+                statusSet = status;
+            }
+        };
+
+        var nextCalled =false;
+        function next() {
+            nextCalled = true;
+        }
+
+        parameterizer().default(req, res, next);
+
+        assert.equal(true, nextCalled);
+    });
+
+
+    it('should invoke next when an accounts scope is passed', function() {
+
+        var req = {
+            params: {
+                scope: 'accounts'
+            }
+        };
+        var statusSent, body;
+        var res = {
+            send: function (status, body) {
+                statusSent = status;
+                bodySent = body;
+            }
+        };
+
+        var nextCalled =false;
+        function next() {
+            nextCalled = true;
+        }
+
+        parameterizer().default(req, res, next);
+
+        assert.equal(true, nextCalled);
```

Comment:
i prefer the spy.
i originally had just an assert but i wanted to explicitly avoid truthiness and be exact.

spy is much clearer that i only care that the function was invoked